examples/move/scratch_off/sources/math.move

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module scratch_off::math {
+
+    /// Hard coding this as 1 billion
+    const MODULO_FACTOR: u256 = 1_000_000_000;
+
+    public fun bytes_to_u256(hashed_beacon: &vector<u8>): u256 {
+        let output = 0;
```

Comment:
I seem to recall that u256 has a `from_le_bytes` method which could be used here instead.

Reply:
I can't find an example of this in the move files. I think it's callable in rust only